ADR/0004-component-image-location.md

```diff
@@ -0,0 +1,105 @@
+# 4. Out-of-the-box image repository for AppStudio users 
+
+Date: Oct 29, 2022 
+
+## Status
+
+Accepted
+
+## Approvers
+
+* Alexey Kazakov
+* Gorkem Ercan
```

Comment:
I want to be an approver on this because it's a security requirement

Reply:
Sure @amfred

Reply:
@sbose78 are you going to add @amfred to fulfill this request?